tortoise/models.py

```diff
@@ -806,6 +809,17 @@ def _set_pk_val(self, value: Any) -> None:
     Can be used as a field name when doing filtering e.g. ``.filter(pk=...)`` etc...
     """
 
+    @classmethod
+    def _validate_relation_type(cls, field_key: str, value: Optional["Model"]) -> None:
```

Comment:
I think `value` should be either not optional or this method should skip validation if it is `None` because `None` is a correct value but with the current implementation, this method is going to fail if `None` is passed.

Reply:
Okay, None is a valid value now.